pages/common/ssh.md

```diff
@@ -31,6 +27,10 @@
 
 `ssh -J {{username}}@{{jump_host}} {{username}}@{{remote_host}}`
 
+- SSH reverse proxy: Expose a localhost port on the remote server (Set `GatewayPorts yes` in sshd_config to allow internet access):
+
+`ssh -NTR {{remote_port}}:localhost:{{local_port}} {{remote_host}}`
```

Comment:
For me this duplicates the `ssh -L {{9999}}:{{example.org}}:{{80}}` example and also a bit inconsistent with it.